lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -1199,3 +1199,16 @@ function dosomething_campaign_node_validate($node, $form, &$form_state) {
     }
   }
 }
+
+  /**
+   * @param object $campaign
+   *   Loaded campaign node to check if given feature is on
```

Comment:
smmmalll thing, but I don't think the `campaign` is actually the node. I think it's the campaign class object from [`Campaign::get`](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_campaign/includes/Campaign.php#L53) I think passing the node object here would break things